satellite/audit/reporter.go

```diff
@@ -58,6 +58,8 @@ func (reporter *Reporter) RecordAudits(ctx context.Context, req Report) (_ Repor
 		zap.Int("failures", len(fails)),
 		zap.Int("offlines", len(offlines)),
 		zap.Int("pending", len(pendingAudits)),
+		zap.Binary("Segment", []byte(path)),
+		zap.String("Segment Path", path),
```

Comment:
would you use stringer here? or when would you use string vs. stringer?

Reply:
`zap.String("", obj.String())` -> `zap.Stringer("", obj)`
Replace it wherever possible for performance reasons. The hole idea of doing this is to not call `obj.String()` depending on the log level.

The Segment Path has not `.String()` methode. We can't chage that one.